exporters/otlp/otlplogs/otlplogshttp/client.go

```diff
@@ -348,7 +348,8 @@ func (d *httpClient) UploadLogs(ctx context.Context, protoLogs []*logspb.Resourc
 			}
 			return newResponseError(resp.Header)
 		default:
-			return fmt.Errorf("failed to send to %s: %s", request.URL, resp.Status)
+			body, _ := io.ReadAll(resp.Body)
+			return fmt.Errorf("failed to send to %s [%s]: %s", request.URL, body, resp.Status)
```

Comment:
Could you post an example of the response? 
Error response body might be quite big due various reasons (long stacktrace and etc) and impact application performance so it better to avoid by default. 

And for the formatted error it is better to put body ( as last parameter)  into separate line if present.

Reply:
Unsure what the best approach is, the error I was getting was a `content-type: text/plain`:

```
couldn't parse labels: 1:7: parse error: unexpected character inside braces: '.'
```

Obviously it gets worse when it is json or something, but not having the error there makes it really hard to figure out what is going wrong.

Reply:
Yes good suggestion let's do that and see how/if we get feedback